docs/codeql/codeql-cli/analyzing-databases-with-the-codeql-cli.rst

```diff
@@ -135,6 +135,60 @@ pack names and use the ``--download`` flag::
 The ``analyze`` command above runs the default suite from ``microsoft/coding-standards v1.0.0`` and the latest version of ``github/security-queries`` on the specified database.
 For further information about default suites, see ":ref:`Publishing and using CodeQL packs <publishing-and-using-codeql-packs>`".
 
+Running a subset of queries in a CodeQL pack
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Additionally, you can include a path at the end of a pack specification to run a subset of queries inside the pack. This applies to any command that locates or runs queries within a pack.
+
+The complete way to specify a set of queries is in the form ``scope/name@range:path``, where:
+
+- ``scope/name`` is the qualified name of a CodeQL pack.
+- ``range`` is a `semver range <https://docs.npmjs.com/cli/v6/using-npm/semver#ranges>`_.
+- ``path`` is a file system path to a single query, a directory containing queries, or a query suite file.
+
+If a ``scope/name`` is specified, the ``range`` and ``path`` are
+optional. A missing ``range`` implies the latest version of the
+specified pack. A missing ``path`` implies the default query suite
+of the specified pack.
+
+The ``path`` can be one of a ``*.ql`` query file, a directory
+containing one or more queries, or a ``.qls`` query suite file. If
+there is no pack name specified, then a ``path`` must be provided,
+and will be interpreted relative to the current working directory
+of the current process.
+
+If a ``scope/name`` and ``path`` are specified, then the ``path`` cannot
+be absolute. It is considered relative to the root of the CodeQL
+pack.
+
+The relevant commands are:
+
+* `codeql database analyze <../manual/database-analyze>`__.
+* `codeql database run-queries <../manual/database-run-queries>`__.
+* `codeql execute queries <../manual/execute-queries>`__.
+* `codeql resolve queries <../manual/resolve-queries>`__.
+
+For example::
+
```

Comment:
```suggestion

```

Reply:
The `::` here is causing the doc check to fail as it's expecting line 166 to be indented. If you commit this suggestion the test _should_ go green.